src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const res = {};
+
+  const estilos = sourceString.split(';').filter(checkEmpty);
+
+  estilos.forEach((estilo) => {
+    const [key, value] = estilo.split(':');
```

Comment:
The `split(':')` method assumes that each style definition contains exactly one colon. If the input string might contain multiple colons in a single style definition, this could lead to incorrect parsing. Consider handling such cases if applicable.